token/cli/src/main.rs

```diff
@@ -101,6 +104,12 @@ pub const MINT_ADDRESS_ARG: ArgConstant<'static> = ArgConstant {
     help: "Address of mint that token account is associated with. Required by --sign-only",
 };
 
+pub const ACCOUNT_OR_MIN_ADDRESS_ARG: ArgConstant<'static> = ArgConstant {
```

Comment:
typo `MIN` but im not sure we want to keep this as per below

Reply:
resolved with e4dc1d8